src/index.ts

```diff
@@ -9,3 +9,37 @@ export {
    Response,
    Router,
 };
+
+// We need to export only types that are used in public interfaces (e.g. those used in
+// concrete classes like Application, Request, Response, Router, exported above).
+export {
+   IRoute,
+   IRouter,
+   PathParams,
+   NextCallback,
+   RouterOptions,
+   RequestProcessor,
+   AnyRequestProcessor,
+   ProcessorOrProcessors,
+   ErrorHandlingRequestProcessor,
+} from './interfaces';
+
+export {
+   CookieOpts,
+   RequestEvent,
+   HandlerContext,
+   LambdaEventSourceType,
+   RequestEventRequestContext,
+} from './request-response-types';
+
+export {
+   StringMap,
+   StringUnknownMap,
+   KeyValueStringObject,
+   StringArrayOfStringsMap,
+} from '@silvermine/toolbox';
```

Comment:
At first glance, it may seem odd to just re-export types from `'@silvermine/toolbox'`, but I did this for two reasons:

1. If we don't, then we'd be forcing clients of `lambda-express` to depend on `'@silvermine/toolbox'` directly if they want access to those types.
2. A client may already depend on `'@silvermine/toolbox'` in their code, but at a different version with a different API. This would allow them to use the types from `'@silvermine/toolbox'` that their version of `lambda-express` uses internally as needed, *and* the types from their version of `'@silvermine/toolbox'`.